test/functional/test_framework/segwit_addr.py

```diff
@@ -27,38 +34,45 @@ def bech32_hrp_expand(hrp):
 
 def bech32_verify_checksum(hrp, data):
     """Verify a checksum given HRP and converted data characters."""
-    return bech32_polymod(bech32_hrp_expand(hrp) + data) == 1
-
+    check = bech32_polymod(bech32_hrp_expand(hrp) + data)
+    if check == 1:
+        return Encoding.BECH32
+    elif check == BECH32M_CONST:
+        return Encoding.BECH32M
+    else:
+        return None
 
-def bech32_create_checksum(hrp, data):
+def bech32_create_checksum(hrp, data, spec):
```

Comment:
It's slightly inconsistent that in the c++ code, this is called `encoding` and is the first argument, and here it's `spec` and the last argument.

Reply:
Changed the test_framework code to also use (encoding,hrp,data) everywhere.